etc/linters.txt

```diff
@@ -0,0 +1,5 @@
+#name=shed,args=--refactor,filetype=all,installer=pip
```

Comment:
This is a PoC for how we can add linters. The installer field is set to eventually making the installation dynamic in the Dockerfile.